### PR TITLE
Added Xulkal ALT Keymap

### PR DESCRIPTION
**Pull request by XScorpion2**
## Description

Massdrop ALT Keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/massdrop/alt/keymaps/xulkal/keymap.c [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+#define EXPAND_LAYOUT(...) LAYOUT(__VA_ARGS__)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT(
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  TD_BSPC, KC_HOME, \
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_END,  \
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGUP, \
+        KC_LSPO, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    TD_COMM, KC_DOT,  KC_SLSH, KC_RSPC,          KC_UP,   KC_PGDN, \
+        KC_LCPO, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, LOWER,   KC_LEFT, KC_DOWN, KC_RGHT  \
```
Comment (fauxpark) [inline keyboards/massdrop/alt/keymaps/xulkal/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/massdrop/alt/keymaps/xulkal/keymap.c [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    [_LOWER] = LAYOUT(
+        _______, RGB_RMOD,RGB_MOD, RGB_TOG, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        RGB_SPI, RGB_SAI, RGB_VAI, RGB_HUI, RESET,   QWERTY,  _______, U_T_AUTO,U_T_AGCR,_______, _______, _______, _______, _______, _______, \
+        RGB_SPD, RGB_SAD, RGB_VAD, RGB_HUD, RGBRST,  GAME,    _______, _______, _______, _______, _______, _______,          _______, _______, \
+        _______, _______, _______, _______, _______, _______, TG_NKRO, _______, _______, _______, _______, _______,          _______, _______, \
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______, _______ \
+    )
+};
+
+#define MODS_SHIFT  (get_mods() & MOD_BIT(KC_LSHIFT) || get_mods() & MOD_BIT(KC_RSHIFT))
```
Comment (fauxpark) [inline keyboards/massdrop/alt/keymaps/xulkal/keymap.c]:
```suggestion
#define MODS_SHIFT  (get_mods() & MOD_MASK_SHIFT)
```

etc.

Comment (drashna) [reply keyboards/massdrop/alt/keymaps/xulkal/keymap.c]: Well, really:
```suggestion
#define MODS_SHIFT  ((get_mods() & MOD_BIT(KC_LSHIFT) == MOD_BIT(KC_LSHIFT)) || (get_mods() & MOD_BIT(KC_RSHIFT) == MOD_BIT(KC_RSHIFT)))
```

Assuming you want to get just right or left shift, I think

Comment (XScorpion2) [reply keyboards/massdrop/alt/keymaps/xulkal/keymap.c]: Eh, it's what's there already in the ALT & CTRL default keymaps.

File: keyboards/massdrop/alt/keymaps/xulkal/keymap.c (hunk 1 above)
Comment (XScorpion2) [reply keyboards/massdrop/alt/keymaps/xulkal/keymap.c]: Eh, it's what's there already in the ALT & CTRL default keymaps.

**Pull request merged**